data_scraper/backup.py

```diff
@@ -105,10 +130,21 @@ def _key_exists(bucket, key):
         bucket.Object(key).load()
     except ClientError as e:
         return int(e.response["Error"]["Code"]) != 404
-    return False
+    return True
 
 
 def _remove_old_files(bucket, prefix):
     old_files = bucket.objects.filter(Prefix=prefix)
     for file in old_files:
         file.delete()
+
+
+def get_bucket_name():
```

Comment:
The convention in Python is to prefix non public (internal) functions with an underscore. So `get_bucket_name()` could be `_get_bucket_name()`